app/Providers/RouteServiceProvider.php

```diff
@@ -13,7 +13,7 @@ class RouteServiceProvider extends ServiceProvider
     /**
      * The path to the "home" route for your application.
      *
-     * This is used by Laravel authentication to redirect users after login.
+     * This is used by Laravel authentication to redirect users after Login.
```

Comment:
?

Reply:
Ook niet aangepast? Ik heb nog nooit aan de providers gezeten. Zal dit niet gebeurd zijn door een pull van develop dat iemand anders dit gedaan heeft? Anders zal er iets gebeurt zijn door PHPStorm bij refactoren van iets ergens anders.

Reply:
Ik maak het lowercase, zeer gek dat dit zomaar verandert is. Ik heb niet aan middlewares of providers gezeten aangezien dat niet mijn deel is.